docs/releaseNotes.yml

```diff
@@ -76,7 +98,7 @@ items:
           redirects</a>, and does not use Envoy's built-in OAuth2 filter.
         docs: https://groups.google.com/g/envoy-announce/c/8nP3Kn4jV7k
   - version: 2.3.0
-    date: '2022-06-06'
+    date: "2022-06-06"
```

Comment:
There's a lot of noise here as `releaseNotes.yml` gets pointlessly reformatted?

Reply:
fingers crossed I resolved all of these. I also disabled the auto-format on save to avoid this.